### ISSUE TITLE
Reposition Search Tree Toggle Button

### ISSUE THREAD

**Issue by andreamah** [microsoft/vscode#161614]
Those who are used to having the collapse button on the far right might accidentally hit the search tree toggle.
<img width="502" alt="image (2)" src="https://user-images.githubusercontent.com/31675041/191996832-bf9baf1e-4ad8-411a-9ccd-0799d3af3c01.png">

Ideally, move this icon to the other side to prevent confusion.

cc @bpasero

**Comment by bpasero** [comment]
To clarify: I think any new position will have an impact to someones muscle memory, that is probably hard to avoid. 

@daviddossett maybe for guidance, I also feel like we have quite exhausted the number of actions for the viewlet, maybe about time to think about introducing a "..." and move some there?

**Comment by andreamah** [comment]
That's true, it is getting a little crowded. What else would be in the `...`? Might seem odd if we don't have plans for anything else in there?

**Comment by andreamah** [comment]
For reference, the button is here for source control (outside of the `...`) so I wonder if people will expect it to be an actual button with an icon in search?
<img width="241" alt="image" src="https://user-images.githubusercontent.com/31675041/191998739-3b2d80e7-e8e4-4194-8ffd-763c672b4b0b.png">

**Comment by bpasero** [comment]
Yeah I like the idea of keeping this in sync with other views, though we probably missed out on keeping some of the buttons in sync, like the refresh button...

**Comment by bpasero** [comment]
Should we remove the "Open Search Editor" action? it is duplicate, there is a link action also from the results:

![image](https://user-images.githubusercontent.com/900690/191999488-ff37803f-bb2c-4dd6-a3bd-050fc9299f7e.png)

**Comment by roblourens** [comment]
Eh, I think you might want to open a search editor before doing a search in the sidebar too.

**Comment by andreamah** [comment]
![image](https://user-images.githubusercontent.com/31675041/192034199-a1e92e57-4ecc-48d3-8b7f-8101a521bbef.png)
I did some thinking and I think it should go here because:

1. It shouldn't go in the far left or right because people probably have muscle memory as refresh/collapse as side-most buttons.
2. It shouldn't go on either side of `clear search results` because the icon for that and `view as list` are too similar, may cause misclicks.

So, by process of elimination, it ends up where I placed it in the photo. Any opinions?
